virt/about-virt.adoc

```diff
@@ -15,4 +15,9 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+1]
 
 You can use {VirtProductName} with the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes], xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShift SDN], or one of the other certified default Container Network Interface (CNI) network providers listed in link:https://access.redhat.com/articles/5436171[Certified OpenShift CNI Plug-ins].
 
+You can check your {VirtProductName} cluster for compliance issues by installing the xref:../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance[Compliance Operator] and running a scan with the xref:../security/compliance_operator/compliance-operator-supported-profiles.adoc#compliance-operator-supported-profiles[`ocp4-moderate` and `ocp4-moderate-node` profiles]. The Compliance Operator uses OpenSCAP, a NIST-certified tool, to scan and enforce security policies provided by the content.
```

Comment:
I'm assuming that NIST is a well-known acronym (even though I wasn't aware of it :)). Does that only apply to the US, though? I wonder if we should include a link to nist.gov to clear up any confusion.

Reply:
Doesn't hurt to add a link.